docs/token_authorization_api.md

```diff
@@ -1,11 +1,28 @@
 # TorchServe token authorization API
 
-## Setup
-1. Download the jar files from [Maven](https://mvnrepository.com/artifact/org.pytorch/torchserve-endpoint-plugin) 
-2. Enable token authorization by adding the `--plugins-path /path/to/the/jar/files` flag at start up with the path leading to the downloaded jar files.
+Torchserve now supports token authorization by default.
```

Comment:
@udaij12 Please create an RFC for this if not already done, before the PR lands

Reply:
What is an RFC Im not aware of what that is.

Reply:
Example: https://github.com/pytorch/serve/issues/3066